src/screens/CardInfo/CardInfoScreen.tsx

```diff
@@ -0,0 +1,216 @@
+import React, { useState } from 'react';
+import { StyleSheet, Text, View, StatusBar, Pressable } from 'react-native';
+
+import useOnPressHandlers from '../../hooks/useOnPressHandlers';
+
+import StyleBase from '../../styles/StyleBase';
+
+import colors from '../../constants/colors';
+
+import { Benefit } from '../../types';
+
+import TopBar from '../../components/Bars/TopBar';
+import TapBar from '../../components/Bars/TapBar';
+import CardTile from '../../components/Cards/CardTile';
+import Tile from '../../components/Miscellaneous/Tile';
+import BenefitList from '../../components/Benefits/BenefitList';
+import CustomButton from '../../components/Miscellaneous/CustomButton';
+import BoxContainer from '../../components/Miscellaneous/BoxContainer';
+
+const SHOW_VIRTUAL = 1;
+
+//temp
+interface ICardInfoScreen {
```

Comment:
To był akurat mój bląd w nazewnictwie tych interfejsów, ale póki ich nie ma dużo pozmieniajmy to na `xScreenProps`, zamiast `IxScreen`. Doczytałem i jednak odeszło się od tej praktyki z dawania oznaczenia przez `I` na komponenty (no chyba, że to interfejs jakiegoś repozytorium, ale to takie coś będziemy robili za parę dni)